src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1362,40 +1432,62 @@ namespace WaterToAirHeatPumpSimple {
+                        VolFlowRate = HeatingAirVolFlowRateDes; // system air flow
+                    }
+                    // cooling design day calculations
                     if (VolFlowRate >= DataHVACGlobals::SmallAirVolFlow) {
                         if (state.dataSize->CurOASysNum > 0) { // coil is in the OA stream
                             MixTemp = state.dataSize->FinalSysSizing(state.dataSize->CurSysNum).OutTempAtCoolPeak;
                             MixHumRat = state.dataSize->FinalSysSizing(state.dataSize->CurSysNum).OutHumRatAtCoolPeak;
                             SupTemp = state.dataSize->FinalSysSizing(state.dataSize->CurSysNum).PrecoolTemp;
                             SupHumRat = state.dataSize->FinalSysSizing(state.dataSize->CurSysNum).PrecoolHumRat;
+                            MixTempSys = MixTemp;
+                            MixHumRat = MixHumRat;
                         } else { // coil is on the main air loop
                             SupTemp = state.dataSize->FinalSysSizing(state.dataSize->CurSysNum).CoolSupTemp;
                             SupHumRat = state.dataSize->FinalSysSizing(state.dataSize->CurSysNum).CoolSupHumRat;
+                            if (VolFlowRate > 0.0) {
+                                OutAirFrac = state.dataSize->FinalSysSizing(state.dataSize->CurSysNum).DesOutAirVolFlow / VolFlowRate;
+                                OutAirFracSys = state.dataSize->FinalSysSizing(state.dataSize->CurSysNum).DesOutAirVolFlow / RatedAirVolFlowRateDes;
+                            } else {
+                                OutAirFrac = 1.0;
+                                OutAirFracSys = OutAirFrac;
+                            }
+                            OutAirFrac = min(1.0, max(0.0, OutAirFrac));
+                            OutAirFracSys = min(1.0, max(0.0, OutAirFracSys));
                             if (state.dataAirSystemsData->PrimaryAirSystems(state.dataSize->CurSysNum).NumOACoolCoils ==
                                 0) { // there is no precooling of the OA stream
                                 MixTemp = state.dataSize->FinalSysSizing(state.dataSize->CurSysNum).MixTempAtCoolPeak;
                                 MixHumRat = state.dataSize->FinalSysSizing(state.dataSize->CurSysNum).MixHumRatAtCoolPeak;
+                                // calculate mixed air temperature with system airflow
+                                MixTempSys = OutAirFracSys * state.dataSize->FinalSysSizing(state.dataSize->CurSysNum).OutTempAtCoolPeak +
+                                             (1.0 - OutAirFracSys) * state.dataSize->FinalSysSizing(state.dataSize->CurSysNum).RetTempAtCoolPeak;
+                                MixHumRatSys = OutAirFracSys * state.dataSize->FinalSysSizing(state.dataSize->CurSysNum).OutHumRatAtCoolPeak +
+                                               (1.0 - OutAirFracSys) * state.dataSize->FinalSysSizing(state.dataSize->CurSysNum).RetHumRatAtCoolPeak;
```

Comment:
Calculate mixed air conditions based on system air flow which is only used as part of an adjustment to the capacity if the system air flow is different than the design air flow rate used to determine the capacity (see calculation of `dHratio` below).